src/server/worker/api_server.go

```diff
@@ -1574,6 +1574,9 @@ func (a *APIServer) getParentHashTree(ctx context.Context, pachClient *client.AP
 	if err != nil {
 		return nil, err
 	}
+	if parentCommitInfo == nil {
+		return ioutil.NopCloser(&bytes.Buffer{}), nil
+	}
```

Comment:
Does this change only affect stats parent hashtrees or is it both? Either way seems like a good change to me.

Reply:
Only stats. This code path is only taken when we are using the parent hashtree as a base for merging (`useParentHashTree` is true).